src/index.ts

```diff
@@ -1,3 +1,4 @@
 import visualizer from "./visualizer";
+import Simulator from "./simulator";
 
-export { visualizer };
+export { visualizer, Simulator };
```

Comment:
should we also export upper case `V` for `visualizer`?

Reply:
visualizer is a function but Simulator is a class, will refactor this in next PR to make both classes